### PR TITLE
Wording Updates

### PR DESCRIPTION
**Pull request by donald-stolz**
These updates are wording and grammar related. I read each article, plugged them into [Grammarly](https://www.grammarly.com/), and made edits off those suggestions
- Copy of #13 with updated branch name to use mirror preview

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ Developer guides contain examples of code, mostly written in JavaScript and Soli
 
 ### Navigating
 
-* If you want an understanding of how **Unstoppable Domains work, its architecture and capabilities**, check out [General Overview](domain-registry-essentials/architecture-overview.md).
-* If you're a **developer** interested in **resolving records for cryptocurrency payments**, check out [Resolving Domain Records](domain-registry-essentials/resolving-domain-records.md).
-* If you're a **developer** interested in **how to resolve our domains in a browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
+- If you want an understanding of how **Unstoppable Domains work, its architecture and capabilities**, check out [General Overview](domain-registry-essentials/architecture-overview.md).
+- If you're a **developer** interested in **resolving records for cryptocurrency payments**, check out [Resolving Domain Records](domain-registry-essentials/resolving-domain-records.md).
+- If you're a **developer** interested in **how to resolve our domains in the browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
```
Comment (mvlabat) [inline src/README.md:25]: Is the indefinite article correct choice here? Right now it's not consistent with the name of the page

File: src/browser-resolution/resolving-domains-in-a-browser.md [hunk 2]
```diff
@@ -15,52 +15,52 @@ Here are some of the end user scenarios that should give an idea which features
 
 ### Distributed Website Browsing
 
-1. Given a blockchain domain has an dweb content identifier record configured \(e.g. IPFS hash\).
-2. When user enters the domain name into a browser address bar.
+1. Given a blockchain domain has a dweb content identifier record configured \(e.g. IPFS hash\).
+2. The user enters the domain name into a browser address bar.
 3. Then the browser resolves the domain and gets the content hash of a domain.
 4. Then the browser retrieves the content by the hash using a related protocol and displays the content.
```
Comment (mvlabat) [inline src/browser-resolution/resolving-domains-in-a-browser.md:21]: Bogdan used such style deliberately, to conform with BDD's "Given-When-Then"

File: src/domain-registry-essentials/architecture-overview.md [hunk 3]
```diff
@@ -60,71 +60,71 @@ A domain owner or an operator may perform the following operations with their do
...
+**Note:** If the owner of 'crypto' top-level domain is set to a [burn address](https://etherscan.io/address/0x000000000000000000000000000000000000dEaD), that means that only direct owners can transfer or burn their second-level domains. To mint second-level domains we use a different mechanism, which doesn't rely on domain ownership: for more information, see the [Minting Controller](architecture-overview.md#minting-controller) section.
 {% endhint %}
 
 ### Alternative Ownership Models
 
 Solidity smart contracts allow other models of managing subdomains to be employed. One of the ways is to use DomainZoneController smart contract, which is supported by Unstoppable Domains team.
 
-DomainZoneController suggests the model of ownership where subdomains can be managed only by their owners. Minting subdomains can be performed using whitelisted accounts, that can be added our renounced by smart contract admins.
+DomainZoneController suggests the model of ownership where subdomains can be managed only by their owners. Minting subdomains can be performed using whitelisted accounts, that can be added or renounced by smart contract admins.
```
Comment (mvlabat) [inline src/domain-registry-essentials/architecture-overview.md:105]: Good catch, that's one of my most annoying mistakes :D I always accidentally confuse "to" with "two" or "too", "our" vs "or" is a painful one too

File: src/domain-registry-essentials/reference.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Records Reference
 
-Records Reference is an overview of all standardized resolver record, that have a defined interpretation. Any other custom records are not forbidden to be set, however their interpretation remains custom to specific clients. See [Resolving Domain Records](resolving-domain-records.md) and [Architecture Overview Resolver](architecture-overview.md#resolver) for more information on domain records.
+Records Reference is an overview of all standardized resolver records that have a defined interpretation. Other custom records are allowed to be set. However, their interpretation remains custom to specific clients. See [Resolving Domain Records](resolving-domain-records.md) and [Architecture Overview Resolver](architecture-overview.md#resolver) for more information on domain records.
```
Comment (mvlabat) [inline src/domain-registry-essentials/reference.md:3]: > remains custom to specific clients
I'm thinking.. is "remains" a lexically correct choice here?

**Comment by mvlabat** [comment]
Thanks for a thorough proofreading! I'm happy to merge this after fixing the first two comments

File: src/README.md [hunk 5]
```diff
@@ -14,22 +14,23 @@ This documentation is written for different audiences and is useful for develope
...
 
 ### Navigating
 
-- If you want an understanding of how **Unstoppable Domains work, its architecture and capabilities**, check out [General Overview](domain-registry-essentials/architecture-overview.md).
-- If you're a **developer** interested in **resolving records for cryptocurrency payments**, check out [Resolving Domain Records](domain-registry-essentials/resolving-domain-records.md).
-- If you're a **developer** interested in **how to resolve our domains in browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
+* If you want an understanding of how **Unstoppable Domains work, its architecture and capabilities**, check out [General Overview](domain-registry-essentials/architecture-overview.md).
+* If you're a **developer** interested in **resolving records for cryptocurrency payments**, check out [Resolving Domain Records](domain-registry-essentials/resolving-domain-records.md).
+* If you're a **developer** interested in **how to resolve our domains in the browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
```
Comment (mvlabat) [inline src/README.md:25]:
```suggestion
* If you're a **developer** interested in **how to resolve our domains in a browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
```

File: src/browser-resolution/resolving-domains-in-a-browser.md (hunk 2 above)
Comment (donald-stolz) [reply src/browser-resolution/resolving-domains-in-a-browser.md:21]: Ok we can change that back, but in I don't like the wording & repetitiveness with these lists. Having the first 3 steps of every list be 90% similar makes me want to skip what I'm reading.

I'd suggest we get rid of steps 1 & 2 everywhere on this page and use a format like this: 
```md
### HTTP Website Browsing

**Given:** A blockchain domain has a DNS record configured.
**When:** The user enters the domain name into a browser address bar.
1. The browser resolves the domain and gets the specified DNS records.
2. Then the browser requests and displays the content using DNS protocol and HTTP protocol.
```

Also "When the user enters the domain name into a browser address bar." Is step 2 for the 1st 3 lists, it seems like it'd be more appropriate to group all 3 of these lists some way and have that step be a given for all of them.

Comment (donald-stolz) [reply src/browser-resolution/resolving-domains-in-a-browser.md:21]: Looking at more of the "Given-When-Then" I didn't see any of them use lists. Most of them were structured like Psedo code:
```
**Scenario**: lorem ipsum
Given:
When:
Then:
And/Then:
```

File: src/domain-registry-essentials/reference.md (hunk 4 above)
Comment (donald-stolz) [reply src/domain-registry-essentials/reference.md:3]: Yes, I think... I tried to find a quick rule but there isn't one. `Remain` & `Remains` are two different words, and looking at the definition for both verb forms, either could be used properly here. Grammarly usually suggested changing to remains and IMO `remains` sounds a bit better

File: src/README.md (hunk 5 above)
Comment (donald-stolz) [reply src/README.md:25]: What's meant by "fix every other article before"? Switching "a" for "the" shouldn't need any other changes

**Comment by mvlabat** [comment]
Trying to stop GitBook from duplicating image assets... Will reopen it a bit later

**Pull request merged**